reflex/reflex.py

```diff
@@ -255,6 +269,38 @@ def export(
         )
 
 
+db_cli = typer.Typer()
+
+
+@db_cli.command()
+def migrate():
+    """Create or update database schema based on app models or existing migration scripts."""
+    # explicitly running `reflex db migrate` overrides config file value
+    get_config().db_migrate = True
+    prerequisites.get_app()
+    prerequisites.check_schema_up_to_date()
+
+
+@db_cli.command()
+def makemigrations():
```

Comment:
It would be nice to have this command accept a `--message` argument so we're able to name migration files for readability. As a user, I'm not sure if I recall not giving any of my migration files a name.